clarity/src/vm/representations.rs

```diff
@@ -455,12 +455,16 @@ pub struct SymbolicExpression {
     pub id: u64,
 
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
```

Comment:
I don't think `Default` is implemented for `Span`.

Reply:
It is here https://github.com/stacks-network/stacks-core/blob/d4da996478a24291d9e154c01ecd6b9c00d991b5/clarity/src/vm/representations.rs#L658

And the default for u32 is 0, so default and Zero are the same

Reply:
Oh, thanks. I must've looked in the wrong branch.